tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+import os
+import pytest
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def pytest_addoption(parser):
+    parser.addoption("--lang", action="store", default="ts")
```

Comment:
I think we can specify the valid choices

```suggestion
    parser.addoption("--lang", action="store", default="ts", choices=["ts", "python"])
```